web/server/www/scripts/codecheckerviewer/CommentView.js

```diff
@@ -98,7 +98,7 @@ function (declare, dom, style, topic, Memory, Observable, ConfirmDialog,
 
       this._message = new ContentPane({
         class   : 'message',
-        content : this.message
+        content : this.message.replace(/(?:\r\n|\r|\n)/g, '<br>')
```

Comment:
You have to do the same thing here:
https://github.com/Ericsson/codechecker/blob/2be579c11046ff7d119f7b4c7ee6f4803376a852/web/server/www/scripts/codecheckerviewer/CommentView.js#L136

This will be fired when you edit a comment and click on to the *Save* button.